changelog/unreleased/kong/fix-rpc-err-msg-checking.yml

```diff
@@ -0,0 +1,4 @@
+message: |
+  More strict parameter checking when generating a RPC error.
```

Comment:
Is there a changelog about this RPC protocol when releasing 3.7.0?

Reply:
Found it: https://github.com/Kong/kong/pull/12320/files#diff-96e1615b63e2a4c46e84dd3d98faa01a5fdfb289a85ef591f539563e1ebc662f

Reply:
> Found it: https://github.com/Kong/kong/pull/12320/files#diff-96e1615b63e2a4c46e84dd3d98faa01a5fdfb289a85ef591f539563e1ebc662f

@chobits I didn't find it in https://github.com/Kong/kong/tree/3.7.0/changelog/3.7.0/kong

Reply:
It seems that the log entry of RPC is removed from 3.7 release.

Reply:
I removed the change log entry too.

Reply:
Cool